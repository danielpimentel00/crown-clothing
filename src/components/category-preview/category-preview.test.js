import { shallow } from "enzyme";
import CategoryPreview from "./category-preview.component";

describe("Category Preview Component", () => {
  it("should match snapshot", () => {
    const mockProps = {
      title: "",
      products: [],
    };
    const wrapper = shallow(<CategoryPreview {...mockProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
