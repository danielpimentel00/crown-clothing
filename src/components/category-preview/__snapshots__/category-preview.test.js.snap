// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Category Preview Component should match snapshot 1`] = `ShallowWrapper {}`;
